### ISSUE TITLE
Errors trying to reset, reinstall Ubuntu.

### ISSUE THREAD

**Issue by rotated8** [microsoft/WSL#4940]
* Your Windows build number: 10.0.18363.657

* What you're doing and what's happening:

Trying to install Ubuntu (non-LTS) from Microsoft Store.
~200 MB download completes.
While installing, an error occurs, Code: 0x80073D05.
Trying to install again will (sometimes) decrease the disk size by the size of the download (i.e, not reusing previous download, not deleting download after failed install).

Previously, I had Ubuntu installed, and attempted to reset it using the Apps & Features --> Ubuntu --> Advanced options --> Reset method. I waited and received a check by the reset button. After which, Ubuntu was no longer a recognized command, and attempting to click on my Start menu short cut nothing happened.

Attempting to use `lxrun /uninstall /full` (and confirming with y) from an admin shell produced Error: 0x80070002.

Unchecking WSL from Windows features, rebooting, checking it again, and rebooting again also did not change the results.

The `%windir%\System32\Tasks\Microsoft\Windows\Windows Subsystem for Linux` folder is empty.

I have run `WSReset.exe`, in case the problem was with the store.

Ubuntu will still show up when searched in the start menu. Right-clicking on it will allow me to tell it to uninstall, and the Start menu says it is, but then nothing happens, and it does not change the above errors.

After the installation fails from the store, `Get-AppxPackage -AllUsers -Name *Ubuntu*` shows a package with PackageFullName of `CanonicalGroupLimited.UbuntuonWindows_1804.2019.521.0_x64__79rhkp1fndgsc`, which I can remove with `Remove-AppxPackage`, but even with the package removed, installation fails.

Ubuntu 18.04 LTS will install from the store just fine.

* What's wrong / what should be happening instead:

Ubuntu should install, and clicking on the tile in the Start menu should start bash.

**Comment by sirredbeard** [comment]
> Reset method. I waited and received a check by the reset button. 

Sometimes it takes a bit longer than the check mark to re-download the Ubuntu image and pre-provision before it becomes launchable again.

Launching in that interim period may have left you in an unusable state.

I would do what you did, remove Ubuntu and the feature via PowerShell, run Disk Cleanup, DISM /Online /Cleanup-Image /RestoreHealth, restart, then reinstall WSL, then reinstall from Store.

**Comment by rotated8** [comment]
Around the time of the previous comment, I ensured that everything was uninstalled as before, using `lxrun`, and `Remove-AppxPackage`, then unselected WSL from Windows features, and rebooted. I then ran Disk Cleanup, and the DISM command from @sirredbeard. After that, I waited.

Since then, my version of Windows has updated- 18363.778 now- so I figured I'd try installing again. I selected the WSL feature, and rebooted, then installed Ubuntu from the Windows store again. 

And everything worked! Thank you @sirredbeard for the help.
